mjolnir/forcefield/local/WormLikeChainOffsetPotential.cpp

```diff
@@ -0,0 +1,11 @@
+#include <mjolnir/forcefield/local/WormLikeChainOffsetPotential.hpp>
+
+#ifndef MJOLNIR_SEPARATE_BUILD
+#error "MJOLNIR_SEPARATE_BUILD flag is required"
+#endif
+
+namespace mjolnir
+{
+    template class WormLikeChainOffsetPotential<double>;
+    template class WormLikeChainOffsetPotential<float>;
```

Comment:
Although it is a nitpicking, normally, we don't have any space here.
```suggestion
template class WormLikeChainOffsetPotential<double>;
template class WormLikeChainOffsetPotential<float>;
```